.jshintignore

```diff
@@ -9,6 +9,7 @@ modules/shortcodes/js/jmpress.js
 modules/shortcodes/js/jmpress.min.js
 modules/shortcodes/js/jquery.cycle.js
 modules/theme-tools/responsive-videos/responsive-videos.min.js
+modules/google-analytics/wp-google-analytics.js
```

Comment:
This isn't being ignored, but it's not your fault.  Can you please add it to https://github.com/Automattic/jetpack/blob/master/gulpfile.js#L449 until we fix this?  Thanks